RoadMap_SwiftUI/RoadMap_SwiftUI/View/AssortmentView.swift

```diff
@@ -0,0 +1,123 @@
+// AssortmentView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Экран выбора товаров
+struct AssortmentView: View {
+    // MARK: - Public properties
+    var body: some View {
+        VStack {
+            headerView
+            Text("Total price $\(viewModel.totalPrice)")
```

Comment:
```suggestion
            Text("Total price $\(viewModel.totalPrice)")
```
Total price $ в константу